e2e/test/scenarios/embedding/interactive-embedding.cy.spec.js

```diff
@@ -772,8 +772,15 @@ describe("scenarios > embedding > full app", () => {
       metric: "Metrics",
     };
 
+    /**
+     *
+     * @param {object} option
+     * @param {boolean} [option.isMultiStageDataPicker]
+     * @param {import("metabase-types/store").InteractiveEmbeddingOptions} [option.qs]
+     */
     function startNewEmbeddingQuestion({
       isMultiStageDataPicker = false,
+      qs,
```

Comment:
I don't love this parameter name. Maybe we can call it `queryString`?

Reply:
that's fair, it's just a passthrough [to Cypress' `visit`](https://docs.cypress.io/api/commands/visit#Arguments) (it uses `qs`), but let's see if that makes things better.

Reply:
addressed in 0977fdd97fbdb7070b8ccadf1f7681b1e31fc7b9